src/libraries/Common/src/System/Security/Cryptography/AesGcm.cs

```diff
@@ -0,0 +1,353 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.Versioning;
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Represents an Advanced Encryption Standard (AES) key to be used with the Galois/Counter Mode (GCM) mode of operation.
+    /// </summary>
+    public sealed partial class AesGcm : IDisposable
```

Comment:
One of the things that made outboxing a little tricky is that the public API was split up in to different partials. This is less fun for Microsoft.Bcl.Cryptography, because it is the "source of truth" for its documentation.

The solution I came up with was to shove the public API surface in to a file that is mostly all partials, and document the partial. Different platforms each fill out the partial, but that way each platform correctly sees that it is documented.

Reply:
On that note, the documentation was back-ported from docs repo. So this required going back and filling in all of the public documentation.

Reply:
> The solution I came up with was to shove the public API surface in to a file that is mostly all partials, and document the partial. 

Yeah, that's consistent with what I did for the cert loader, and think is our path forward... given our split builds and the docs source-of-truth location being in the code repository.

It'll also help with keeping attributes in sync (like SupportedPlatform)